frontend/src/scenes/session-recordings/player/playerMetaLogic.ts

```diff
@@ -47,15 +47,15 @@ export const playerMetaLogic = kea<playerMetaLogicType>({
 
                 const currIndex = findLastIndex(
                     snapshots,
-                    (s: eventWithTime) => s.timestamp < currentEpochTime && 'width' in s.data
+                    (s: eventWithTime) => s.timestamp < currentEpochTime && (s.data as any).width
```

Comment:
being very nitty this doesn't find events with width 0... 

Maybe we don't care about them for playback anyway 🤣